### PR TITLE
Testing aragon UI 2

### PR DESCRIPTION
**Pull request by Rekard0**
- import aragon ui v2
- add aragon ui main to home page
- new Create Dao flow

### CODE REVIEW

File: packages/govern-console/src/containers/CreateDao/CreateDaoBasicInfo.tsx [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+    tokenName,
+    tokenSymbol,
+    tokenAddress,
+    tokenMintAmount,
+    isProxy,
+  } = basicInfo;
+
+  const updateBasicInfo = (
+    indexType:
+      | 'daoIdentifier'
```
Comment (yuetloo) [inline packages/govern-console/src/containers/CreateDao/CreateDaoBasicInfo.tsx]: we can make this cleaner if we define a type like this : https://github.com/aragon/upgrade/blob/master/src/token-info/types.ts

File: packages/govern-console/src/containers/CreateDao/CreateDaoCollateral.tsx [hunk 2]
```diff
@@ -0,0 +1,312 @@
...
+    isChallengeNewDaoToken,
+    isAnyAddress,
+    executionAddressList,
+  } = collaterals;
+
+  const updateCollaterals = (
+    indexType:
+      | 'scheduellAddress'
+      | 'scheduellAmount'
+      | 'isScheduellNewDaoToken'
```
Comment (yuetloo) [inline packages/govern-console/src/containers/CreateDao/CreateDaoCollateral.tsx]: typo for scheduleAddress, scheduleAmount, isScheduleNewDaoToken.. and would like to see a type defined for indexType

File: packages/govern-console/src/containers/CreateDao/CreateDaoCollateral.tsx [hunk 3]
```diff
@@ -0,0 +1,312 @@
...
+    value: any,
+  ) => {
+    console.log('updateCollaterals', indexType, value);
+    const newCollaterals: ICreateDaoCollaterals = { ...collaterals };
+    (newCollaterals[indexType] as any) = value;
+    setCollaterals(newCollaterals);
+  };
+
+  return (
+    <Grid layout={true}>
```
Comment (yuetloo) [inline packages/govern-console/src/containers/CreateDao/CreateDaoCollateral.tsx:48]: If you can break this into smaller component, will make this code cleaner and easier to read.

File: packages/govern-console/src/containers/CreateDao/CreateDaoCollateral.tsx [hunk 4]
```diff
@@ -0,0 +1,312 @@
...
+                }}
+              >
+                Next Step
+              </Button>
+            }
+          />
+        </Box>
+      </GridItem>
+      <GridItem
+        gridRow={layoutName === 'large' ? '1' : undefined}
```
Comment (yuetloo) [inline packages/govern-console/src/containers/CreateDao/CreateDaoCollateral.tsx:284]: does layoutName changes based on the device size?

File: packages/govern-console/src/containers/CreateDao/Shared.tsx [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+  isProxy: 'Use Proxy',
+
+  executionDelay: 'Execution Delay',
+  isRuleFile: 'Use Rule File',
+  ruleFile: 'Rule File',
+  ruleText: 'Rule Text',
+  resolver: 'Resolver',
+
+  scheduellAddress: 'Scheduell Address',
+  scheduellAmount: 'Scheduell Amount',
```
Comment (yuetloo) [inline packages/govern-console/src/containers/CreateDao/Shared.tsx]: typo on the Schedule Address/Amount

**Review by yuetloo — APPROVED** [review]

File: packages/govern-console/src/containers/CreateDao/CreateDaoCollateral.tsx (hunk 4 above)
Comment (Rekard0) [reply packages/govern-console/src/containers/CreateDao/CreateDaoCollateral.tsx:284]: layoutName changes bases on windows width size, regardless of device size,
so for example: if the device is Ipad Pro, we probably get layout name 'medium' if the device is in portrait orientation, and 'large' if orientation is at landscape

File: packages/govern-console/src/containers/CreateDao/CreateDaoCollateral.tsx (hunk 3 above)
Comment (Rekard0) [reply packages/govern-console/src/containers/CreateDao/CreateDaoCollateral.tsx:48]: yes, i have plans to do the split once we have the switcher componer ready

**Pull request merged**